docs/concepts/app-anatomy.md

```diff
@@ -0,0 +1,214 @@
+# Anatomy of an SDK Application
+
+## Pre-requisite reading
+
+- [High-level overview of the architecture of an SDK application](../intro/sdk-app-architecture.md)
+- [Cosmos SDK design overview](../intro/sdk-design.md)
+
+## Synopsis
+
+This document describes the core parts of a Cosmos SDK application. The placeholder name for this application will be `app`.
+
+- [Node Client](#node-client)
+- [Core Application File](#core-application-file)
+- [Modules](#modules)
+- [Interfaces](#interfaces)
+- [Dependencies and Makefile](#dependencies-and-makefile)
+
+The core parts listed above will generally translate to the following directory tree:
+
+```
+./app
+├── cmd/
+│   ├── appd
+│   └── appcli
+├── app.go
+├── x/
+│   ├── auth
+│   └── bank
+├── go.mod
```

Comment:
```suggestion
├── go.mod
├── go.sum
```

Reply:
Agree with @rigelrozanski, I think `go.mod` is sufficient. Maybe it's implicit, but does not hurt to have it nonetheless

Reply:
If we're stating what a standard application _should_ look like as far as structure, then I do think it should be included.